traits/ctraits.c

```diff
@@ -4253,6 +4259,37 @@ _trait_comparison_mode(trait_object *trait, PyObject *args)
     return Py_None;
 }
 
+/*-----------------------------------------------------------------------------
+|  getter for trait comparison mode
++----------------------------------------------------------------------------*/
+
+static PyObject *
+_get_trait_comparison_mode_int(trait_object *trait, PyObject *Py_UNUSED(ignored))
+{
+    int i_comparison_mode;
+
+    unsigned long compare_flag = trait->flags & TRAIT_COMPARE_MASK;
```

Comment:
nit: suggest `unsigned int` instead of `unsigned long` to match the types of `trait->flags` and `TRAIT_COMPARE_MASK`.